src/Entries/Entry.php

```diff
@@ -76,11 +99,11 @@ public function origin($origin = null)
             return $this->origin;
         }
 
-        if (! $this->model->origin) {
-            return null;
+        if (! $this->model?->origin_id) {
+            return;
```

Comment:
This chunk (probably among others) should resolve #44, right?

Reply:
Yep!